tests/discussions.test.js

```diff
@@ -0,0 +1,364 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
const chaiHttp = require("chai-http");
```